src/main/java/org/traccar/protocol/HuabaoProtocolDecoder.java

```diff
@@ -52,6 +52,8 @@ public HuabaoProtocolDecoder(Protocol protocol) {
     public static final int MSG_TERMINAL_REGISTER_RESPONSE = 0x8100;
     public static final int MSG_TERMINAL_CONTROL = 0x8105;
     public static final int MSG_TERMINAL_AUTH = 0x0102;
+    public static final int MSG_TERMINAL_PARAM_QUERY_REPORT = 0x0107;
```

Comment:
I don't see this one in the document.

Reply:
The B50 device was sending the 0x0107 messages, but no position messages. I queried with the manufacturer. They replied with the following message, "0107 message type customers can not parse, but must reply to the generic answer, otherwise the device will think the server is abnormal and will not send location packets". The provided the following document, 
[0107 type message.pdf](https://github.com/traccar/traccar/files/7041642/0107.type.message.pdf)

Reply:
In that case it probably makes sense to reorganize the code to reply to all messages with generic response.